tests/migrations/test_commands.py

```diff
@@ -492,6 +492,40 @@ def test_showmigrations_plan_no_migrations(self):
         call_command('showmigrations', format='plan', stdout=out, verbosity=2, no_color=True)
         self.assertEqual('(no migrations)\n', out.getvalue().lower())
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_initial_squash"})
+    def test_showmigrations_list_squashed(self):
+        """
+            Ensures a migration that doesn't have an applied time stamp can still be
+            supported with high verbosity
+        """
+        out = io.StringIO()
+        call_command("showmigrations", format='list', stdout=out, verbosity=2)
```

Comment:
Use `no_color=True` to about matching against escape sequences. It looks like `verbosity=2` is also unnecessary?

```suggestion
        call_command("showmigrations", format='list', stdout=out, no_color=True)
```

Reply:
Thank your for pointing out `no_color`, that was very helpful.

`Verbosity=2` is necessary because that is the code path I want to trigger to ensure this pull request is fixing the encountered error associated to setting verbosity to `2`